docs/data/charts/legend/legend.md

```diff
@@ -60,6 +60,12 @@ Style applied by other means will not be taken into account.
 Which can lead to label overflow.
 :::
 
+### Rounded symbol
+
+To create a rounded symbol, use the `sx` prop to style the symbol class directly. Import `legendClasses.mark` to target the symbol, and set the `ry` property to control the symbol's corner radius.
```

Comment:
```suggestion
### Rounded symbol

To create a rounded symbol, use the `legendClasses.mark` to apply CSS on marks.
Notice that SVG `rect` uses `ry` property to control the symbol's corner radius instead of `border-radius`.
```

I propose to be more generic, because `sx` is not the only way we can pass CSS to the component

Reply:
Updated